meetings/2025-02-05.md

```diff
@@ -28,18 +35,89 @@
+  I think we need to have at least Jesse and Jackson Morgan.
+* Rahul: Michiel de Jong has collated work on this, you might want to include it as well. https://pdsinterop.org/conventions/
+* Jesse: You should look at the [solid/roadmap repo](https://github.com/solid/roadmap)
+
+ACTION: SR to prepare a short document with requirements and organize the STM meeting, with a poll to ensure people can be present.
+
+* RG: Are you still going forward with that grant proposal?
+* Jesse: There was a bounty proposal system, reviewers didn't understand how shapes work in Solid. I spoke with NLnet at FOSDEM, they are keen to support Solid and we could rework the grant and get it fast forwarded. We can discuss it as part of the STM.
+* eP: Will have a call with NLnet soon about a grant I asked before Oct 1st and it moved to phase 2, I will mention this to them.
+
+### Insertion with Blank node with N3 Patch
+
+https://github.com/CommunitySolidServer/CommunitySolidServer/issues/1968
+
+https://github.com/solid/specification/issues/704
+
+* RG: Context: Migrating from NSS to CSS. NSS supports SPARQL update and N3 Patch, but CSS only supports N3 Patch. I don't know the context of the discussion, but want to know what I can do to make SolidOS work as it may be broken with only N3 Patch
+* Jesse: This seems like a discussion for the N3 CG, not the Solid CG.
+* eP: N3 is not so mature, can we really use it as a dependency?
+* Jesse: N3 is thinking about becoming a WG.
+* eP: N3-patch only exist within the Solid spec, so what will happen with the new WG ?
+* RG: We are not telling the WG to include this, but we are making them aware that there are some problems (?)
+* Jesse: If you have a clear solution in mind, no one is going to object that, and the WG will take this as a valuable input.
+* ...
+* eP: If someone has a clear idea about why blank nodes should be allowed with N3 patch, they should create a PR so we can discuss on concrete terms with the community. And maybe warn the WG about this change.
+* Jesse: My proposal is to delete the "MUST NOT"
+* HZ: Minimal change could change *MUST NOT* to *SHOULD NOT*
+
+ACTION: Jesse will create a PR removing that line, he will link to this discussion.
+
+### Misc
 
+* SR: I want to post discussion about profile management. It seems that Solid didn't think about it. The profiles are public, there is also a need for private profiles. If you share the full profile it can be problematic. The profile is the most sensitive data on the pod.
+* JW: There is a notion of extended profiles that are used by implementaiton. In public profile there are minimal information, there is a pointer to extended profile which are access protected.
+* SR: It is not only about public/private but about fine grained access. For now it is a single resource.
+* HZ: I think we will need to discuss views and contexts. 
```

Comment:
```suggestion
* SR: I want to post discussion about profile management. It seems that Solid didn't think about it. All profiles are now public. There is also a need for private profiles. If you share the full profile, it can be problematic. The profile is the most sensitive data on the pod.
* JW: There is a notion of extended profiles that are used by implementation. In public profiles, there is minimal information; there is a pointer to extended profiles, which are access protected.
* SR: It is not only about public/private, but also about fine grained access. For now, it is a single resource.
* HZ: I think we will need to discuss views and contexts. 
```